src/content/blog/what’s-new-with-the-filecoin-plus-notary-election-and-filecoin-day-highlights-from-labweek23.md

```diff
@@ -84,7 +83,7 @@ FF Co-Founding Officer Clara Tsao discussed why the future of AI and enterprise
 
 **Crossing the Streams: A Ghostbusters-themed Approach to Scaling the Filecoin Ecosystem**
 
-<div style="text-align: center;">
+<figure>
```

Comment:
If there's not `figcaption` - There's no need to wrap the `iframe` in such a tag.